src/component/feature/FeatureView/FeatureSettings/FeatureSettingsInformation/FeatureSettingsInformation.tsx

```diff
@@ -0,0 +1,62 @@
+import { Typography } from '@material-ui/core';
+import { Edit } from '@material-ui/icons';
+import { useHistory } from 'react-router-dom';
+import useFeature from '../../../../../hooks/api/getters/useFeature/useFeature';
+import PermissionIconButton from '../../../../common/PermissionIconButton/PermissionIconButton';
+import { UPDATE_FEATURE } from '../../../../providers/AccessProvider/permissions';
+
+interface IFeatureSettingsInformationProps {
+    projectId: string;
+    featureId: string;
+}
+
+export const FeatureSettingsInformation = ({
+    projectId,
+    featureId,
+}: IFeatureSettingsInformationProps) => {
+    const { feature } = useFeature(projectId, featureId);
+    const history = useHistory();
+
+    const onEdit = () => {
+        history.push(`/projects/${projectId}/features/${featureId}/edit`);
+    };
+
+    return (
+        <>
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+                <Typography variant="h5" component="div">
```

Comment:
We don't use this font size anywhere else in the application. Please use the defined font types in the theme here. You can use `theme.fontSizes.mainHeader`.